builtin/providers/aws/provider.go

```diff
@@ -156,76 +156,78 @@ func Provider() terraform.ResourceProvider {
-			"aws_route53_delegation_set":       resourceAwsRoute53DelegationSet(),
-			"aws_route53_record":               resourceAwsRoute53Record(),
-			"aws_route53_zone_association":     resourceAwsRoute53ZoneAssociation(),
-			"aws_route53_zone":                 resourceAwsRoute53Zone(),
-			"aws_route53_health_check":         resourceAwsRoute53HealthCheck(),
-			"aws_route_table":                  resourceAwsRouteTable(),
-			"aws_route_table_association":      resourceAwsRouteTableAssociation(),
-			"aws_s3_bucket":                    resourceAwsS3Bucket(),
-			"aws_s3_bucket_object":             resourceAwsS3BucketObject(),
-			"aws_security_group":               resourceAwsSecurityGroup(),
-			"aws_security_group_rule":          resourceAwsSecurityGroupRule(),
-			"aws_spot_instance_request":        resourceAwsSpotInstanceRequest(),
-			"aws_sqs_queue":                    resourceAwsSqsQueue(),
-			"aws_sns_topic":                    resourceAwsSnsTopic(),
-			"aws_sns_topic_subscription":       resourceAwsSnsTopicSubscription(),
-			"aws_subnet":                       resourceAwsSubnet(),
-			"aws_volume_attachment":            resourceAwsVolumeAttachment(),
-			"aws_vpc_dhcp_options_association": resourceAwsVpcDhcpOptionsAssociation(),
-			"aws_vpc_dhcp_options":             resourceAwsVpcDhcpOptions(),
-			"aws_vpc_peering_connection":       resourceAwsVpcPeeringConnection(),
-			"aws_vpc":                          resourceAwsVpc(),
-			"aws_vpc_endpoint":                 resourceAwsVpcEndpoint(),
-			"aws_vpn_connection":               resourceAwsVpnConnection(),
-			"aws_vpn_connection_route":         resourceAwsVpnConnectionRoute(),
-			"aws_vpn_gateway":                  resourceAwsVpnGateway(),
+			"aws_app_cookie_stickiness_policy":  resourceAwsAppCookieStickinessPolicy(),
+			"aws_autoscaling_group":             resourceAwsAutoscalingGroup(),
+			"aws_autoscaling_notification":      resourceAwsAutoscalingNotification(),
+			"aws_autoscaling_policy":            resourceAwsAutoscalingPolicy(),
+			"aws_cloudwatch_metric_alarm":       resourceAwsCloudWatchMetricAlarm(),
+			"aws_customer_gateway":              resourceAwsCustomerGateway(),
+			"aws_db_instance":                   resourceAwsDbInstance(),
+			"aws_db_parameter_group":            resourceAwsDbParameterGroup(),
+			"aws_db_security_group":             resourceAwsDbSecurityGroup(),
+			"aws_db_subnet_group":               resourceAwsDbSubnetGroup(),
+			"aws_dynamodb_table":                resourceAwsDynamoDbTable(),
+			"aws_ebs_volume":                    resourceAwsEbsVolume(),
+			"aws_ecs_cluster":                   resourceAwsEcsCluster(),
+			"aws_ecs_service":                   resourceAwsEcsService(),
+			"aws_ecs_task_definition":           resourceAwsEcsTaskDefinition(),
+			"aws_eip":                           resourceAwsEip(),
+			"aws_elastic_beanstalk_application": resourceAwsElasticBeanstalkApplication(),
```

Comment:
Indeed... it seems that brevity has been the focus in existing names anyway, with e.g. ``eip`` instead of ``elastic_ip``... so ``app`` here would be hardly the worst example of abbreviating things.